go-controller/pkg/node/helper_linux.go

```diff
@@ -36,6 +36,9 @@ func getDefaultGatewayInterfaceDetails(gwIface string) (string, []net.IP, error)
 		if err != nil {
 			return "", gatewayIPs, err
 		}
+		if intfIPv6Name == "" {
```

Comment:
Question: Does it make sense in our implementation to [`return "", net.IP{}, nil`](https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/node/helper_linux.go#L129) from `getDefaultGatewayInterfaceByFamily` when there is nothing found? Or should it just be an error since we are not able to find the default gw?

Reply:
No, it doesnt make sense. Let me update.

Reply:
In case when there is no gw ip we are adding empty IPs to `gatewayIPs`.
For IPv6 it would get used here: https://github.com/ovn-org/ovn-kubernetes/blob/72345e0b3551603df72627702ec60c70c7885221/go-controller/pkg/node/gateway_init.go#L150
IIUC `utilnet.IsIPv6` will return false on an empty IP, meaning that `getGatewayNextHops` will return an empty list of IPs for IPv6 cluster?